lib/gallerist/app/helpers.rb

```diff
@@ -17,8 +17,12 @@ def link_to(obj, classes = nil)
 
     case obj
     when Gallerist::Album
-      link = '<div class="key-photo"><img data-layzr="/thumbs/%d"></div> %s' %
-              [ obj.key_photo.id, obj.name ]
+      if obj.key_photo
+        link = '<div class="key-photo"><img data-layzr="/thumbs/%d"></div> %s' %
+                [ obj.key_photo.id, obj.name ]
+      else
+        link = '<div class="key-photo"></div> %s' % [ obj.name ]
+      end
```

Comment:
This seems to be a change introduced in the Photos app with Yosemite 10.10.4. I have an error with one of my libraries here, too.

I don’t know if just having no key photo is the right way to go, but that seems OK for now.